app.rb

```diff
@@ -6,6 +6,10 @@
 require "dotenv"
 require "text"
 require "sanitize"
+require "pycall"
+
+# TODO: figure out how to install ftfy in the context of this project/deployment, see https://github.com/rspeer/python-ftfy
```

Comment:
This was done in https://github.com/mduleone/trebekbot/pull/10/commits/de32af90f93897f774081a8a097ef675a498a88b and https://github.com/mduleone/trebekbot/pull/10/commits/5b13836b4eb497436123e488db6e144f493cb7b1 as described in https://github.com/mrkn/pycall.rb/tree/f7cde0201e32328907e5613aadbca52c6b38f5ff#deploying-on-heroku

```suggestion
```